src/Controller/CaseController.php

```diff
@@ -158,12 +161,22 @@ public function summary(BoardRepository $boardRepository, CaseEntity $case, Note
         $communications = $digitalPostRepository->findByEntity($case, [], ['createdAt' => Criteria::DESC], $numberOfShownItems);
 
         $suitableBoards = $boardRepository->findDifferentSuitableBoards($case->getBoard());
+        // Check if case is deletable
+        $isDeletable = true;
+        // Case is not deletable if case is on active agenda
+        foreach ($case->getAgendaCaseItems() as $agendaCaseItem) {
+            assert($agendaCaseItem instanceof AgendaCaseItem);
```

Comment:
You could add a `@return Collection|AgendaCaseItem[]` annotation on `CaseEntity::getAgendaCaseItems()` to help the static code analysis (and skip the `assert`ion here).

Reply:
Added return annotation on `CaseEntity::getAgendaCaseItems()`.